### ISSUE TITLE
Some project prefs dont apply without reload of project

### ISSUE THREAD

**Issue by Naatan** [Komodo/KomodoEdit#93]
Go into project prefs, change exclude rules. Upon saving the rules dont apply yet to eg. Places and Commando, its not until you reload the project that they apply.

**Comment by cgchoffman** [comment]
This is odd.  I was looking for another issue and found this one.  This is working fine for me on Windows.
- Edit Project Preferences > Files to exclude
- apply/ok
- open commando, path or pattern excluded is not visible

NOTE: Places never updates for me.  Even closing and reopening the project doesn't exclude the specified pattern.

**Comment by Naatan** [comment]
Seems I logged this one twice, this was fixed in #129

Not sure about your NOTE - may just be something about the particular pattern you specified? Log a new bug if you can repro please.
